gcore/src/prog.rs

```diff
@@ -35,7 +45,24 @@ mod sys {
     }
 }
 
-/// Creates a new program and returns its address.
+/// Same as [`create_program_with_gas`], but without gas limit
```

Comment:
```suggestion
/// Same as [`create_program_with_gas`], but without explicit gas limit.
```

Also need somehow ref docs to how gasless messaging works